src/components/lib/SideToggleComponent/SideToggleComponent.js

```diff
@@ -0,0 +1,15 @@
+export default {
+  data () {
+    return {
+      isClicked: false
+    }
+  },
+  created: function () {
+    this.isClicked = false
```

Comment:
You don't need this because you already declared `isClicked` in `data` function with same `false` value